desktop/test/run-visdiff.js

```diff
@@ -174,9 +174,6 @@ const commitRange = process.argv[2]
   .split(/\.{2,3}/)  // Travis gives us ranges like START...END
   .map(s => s.substr(0, 12))  // trim the hashes a bit for shorter paths
 
-// we want to compare with commit prior to the first in the range
-commitRange[0] += '^'
```

Comment:
Re: https://github.com/keybase/client/pull/2822#discussion_r64287715

GitHub should already be giving us the base commit, which gets sent to Travis. I'm testing this theory as we speak.